sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -98,14 +98,24 @@ DogListProperty dog = new DogListProperty
     Weight = 1.1,
     FoodConsumed = { "kibble", "egg", "peanut butter" },
 };
+
+//stj example
 string json = JsonSerializer.Serialize(dog);
+
+//modelSerializer example
+Stream stream = ModelSerializer.Serialize(dog);
```

Comment:
It would be helpful to document how these are different and when a caller would chose one approach vs another.

Qn - why return stream vs. BinaryData?

Reply:
There will be a few overloads we end up with here it one by one or the other.  For demo purposes we are only showing one to minimize throw away code